ranger/default.go

```diff
@@ -106,7 +107,8 @@ const (
 )
 
 var (
-	defaultBaseURL = "http://" + DefaultHost + DefaultPort
+	defaultAssetsURL = "/"
```

Comment:
I stuck with the existing default assets URL which is the root path.  As setting it to the defaultBaseURL with a scheme and host will cause CORS issues for `connect.xyplanningnetwork.com`.  While not all CDN's can be configured to include a CORS header, the one I think we're most likely to use, Cloufront, does as does the Vite dev server.